lambda/src/app.js

```diff
@@ -111,3 +107,67 @@ exports.lambdaHandler = async (event, context) => {
     };
   }
 };
+
+async function rekognizeLabels(imageBytes) {
```

Comment:
We're mixing arrow notation and `function` functions in this file...

Reply:
Which company's style guide do we want to adhere to?

- Airbnb says [use named functions at the top-level of files](https://github.com/airbnb/javascript#functions--declarations)
- Google implies either is fine, [but seems to prefer arrow functions](https://google.github.io/styleguide/jsguide.html#features-functions-top-level-functions)